### DISCUSSION TITLE
Welcome to terraform-provider-terracurl Discussions!

### DISCUSSION THREAD

**Discussion by devops-rob** [devops-rob/terraform-provider-terracurl#76] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by bschaatsbergen** [comment]
Hi Rob,

What do you think about this feature request: https://github.com/devops-rob/terraform-provider-terracurl/issues/43? I believe it could be a good way to stand out from the http provider.
